app/styles/_section-list.scss

```diff
@@ -61,3 +51,8 @@
     margin-top: 0;
   }
 }
+
+// Utilities
+.bc-u-max-width-full {
```

Comment:
please put utilities into a different file as they could be used elsewhere in the app(s) so are not component specific

Reply:
Oh, I genuinely thought I had

Reply:
done